### DISCUSSION TITLE
引用依赖与其他第三方依赖冲突，请问这个怎么处理或解决？

### DISCUSSION THREAD

**Discussion by dh3922** [Tencent/libpag#1649] [Q&A]
引用依赖与其他第三方依赖冲突，请问这个怎么处理或解决？
安卓平台，第三方依赖为创蓝云智人脸识别
代码如下：【> A failure occurred while executing com.android.build.gradle.internal.tasks.CheckDuplicatesRunnable
   > Duplicate class a.a found in modules jetified-face_sdk_v1.2.0.2-runtime (face_sdk_v1.2.0.2.aar) and jetified-libpag-4.2.80-runtime (com.tencent.tav:libpag:4.2.80)
     Duplicate class a.b found in modules jetified-face_sdk_v1.2.0.2-runtime (face_sdk_v1.2.0.2.aar) and jetified-libpag-4.2.80-runtime (com.tencent.tav:libpag:4.2.80)】

**Comment by kevingpqi123** [comment]
为了提高对接效率，这类问题建议提供完整的复现 demo 工程

**Comment by kevingpqi123** [comment]
一周未回复，暂时关闭
